cache.js

```diff
@@ -54,12 +54,25 @@ let put = function (key, value) {
  * @param {string} key key whose mapping is to be removed from the map
  * @returns {(*|null)} the previous value associated with key or null if there was no mapping for key
  */
-let remove = function (key) {
+const remove = function (key) {
     return cache.remove(key);
-}
+};
+
+/**
+ * Checks whether the mapping for a key from this map exists.
+ * 
+ * @memberof cache
+ * @param {string} key key whose mapping is to be checked in the map
+ * @returns {boolean} whether the key has a mapping therefore whether it exists
+ */
+const exists = function (key) {
+    return (get(key) === null) ? false : true;
```

Comment:
wouldn't `return get(key) !== null` suffice to return ?

Reply:
Yes, it is the same (personally, I prefer the ternary if as it is more clear for me).